masterlist.yaml

```diff
@@ -314,6 +314,14 @@ common:
     name: 'skse/plugins/po3_SpellPerkItemDistributor.dll'
     display: '[Spell Perk Item Distributor](https://www.nexusmods.com/skyrimspecialedition/mods/36869/)'
 
+  - &DAR
+    name: 'skse/plugins/DynamicAnimationReplacer.dll'
+    display: '[Dynamic Animation Replacer](https://www.nexusmods.com/skyrimspecialedition/mods/33746)'
+
+  - &OAR
+    name: 'skse/plugins/OpenAnimationReplacer.dll'
+    display: '[Open Animation Replacer](https://www.nexusmods.com/skyrimspecialedition/mods/92109)'
+
```

Comment:
This part can be removed, as the other two PRs (https://github.com/loot/skyrimse/pull/2855 and https://github.com/loot/skyrimse/pull/2858) already add this.